go-controller/pkg/ovn/namespace.go

```diff
@@ -331,6 +331,12 @@ func (oc *Controller) deleteNamespace(ns *kapi.Namespace) {
 		return
 	}
 	defer nsInfo.Unlock()
+
+	klog.V(5).Infof("Deleting Namespaces's NetworkPolicy entities")
```

Comment:
Typo: Is this a single namespace?  The message should either be Namespace's or if it does multiple Namespaces'

Reply:
It's a single namespace, thanks for catching that!